source/_integrations/adaptive_lighting.markdown

```diff
@@ -0,0 +1,257 @@
+  required: false
+  default: 1
+  type: time
+transition:
+  description: How long the transition is when the lights change, in seconds.
+  required: false
+  default: 45
+  type: integer
+interval:
+  description: How often to adapt the lights, in seconds.
+  required: false
+  default: 90
+  type: integer
+min_brightness:
+  description: The minimum percent of brightness to set the lights to.
+  required: false
+  default: 1
+  type: integer
+max_brightness:
+  description: The maximum percent of brightness to set the lights to.
+  required: false
+  default: 100
+  type: integer
+min_color_temp:
+  description: The warmest color temperature to set the lights to, in Kelvin.
+  required: false
+  default: 2000
+  type: integer
+max_color_temp:
+  description: The coldest color temperature to set the lights to, in Kelvin.
+  required: false
+  default: 5500
+  type: integer
+sleep_brightness:
+  description: Brightness of lights while the sleep mode is enabled.
+  required: false
+  default: 1
+  type: integer
+sleep_color_temp:
+  description: Color temperature of lights while the sleep mode is enabled.
+  required: false
+  default: 1000
+  type: integer
+sunrise_time:
+  description: Override the sunrise time with a fixed time.
+  required: false
+  type: time
+sunrise_offset:
+  description: Change the sunrise time with a positive or negative offset.
+  required: false
+  default: 0
+  type: time
+sunrise_time:
+  description: Override the sunset time with a fixed time.
+  required: false
+  type: time
+sunset_offset:
+  description: Change the sunset time with a positive or negative offset.
+  required: false
+  default: 0
+  type: time
+only_once:
+  description: Whether to keep adapting the lights (false) or to only adapt the lights as soon as they are turned on (true).
+  required: false
+  default: false
+  type: boolean
+take_over_control:
+  description: If another source calls `light.turn_on` while the lights are on and being adapted, disable Adaptive Lighting.
+  required: false
+  default: true
+  type: boolean
+detect_non_ha_changes:
+  description: Whether to detect state changes and stop adapting lights, even not from `light.turn_on`. Needs `take_over_control` to be enabled. Note that by enabling this option, it calls 'homeassistant.update_entity' every 'interval'!
+  required: inclusive
+  default: false
+  type: boolean
+{% endconfiguration %}
+
+Full example:
+
+```yaml
+# Example configuration.yaml entry
+adaptive_lighting:
+- name: "default"
+  lights: []
+  adapt_brightness: true
+  adapt_color_temp: true
+  adapt_rgb_color: true
+  prefer_rgb_color: false
+  transition: 45
+  initial_transition: 1
+  interval: 90
+  min_brightness: 1
+  max_brightness: 100
+  min_color_temp: 2000
+  max_color_temp: 5500
+  sleep_brightness: 1
+  sleep_color_temp: 1000
+  sunrise_time: "08:00:00"  # override the sunrise time
+  sunrise_offset: "00:15:00"
+  sunset_time:
+  sunset_offset: -1800  # in seconds
```

Comment:
```suggestion
  sunrise_offset:
  sunset_time:
  sunset_offset: 1800  # in seconds or '00:15:00'
```

because it doesn't make sense to have an offset and manually defined sunrise.